scout/adapter/mongo/event.py

```diff
@@ -210,6 +208,33 @@ def events(
 
         return self.event_collection.find(query).sort("created_at", pymongo.DESCENDING)
 
+    def events_by_variant_id(
```

Comment:
At least locally, it seems the planner agrees adding category is no shortcut:
![Screenshot 2022-06-20 at 11 52 26](https://user-images.githubusercontent.com/758570/174576399-e2b2b1de-0b16-4cb7-b4e3-0bcc7a623718.png)
I don't know if it would act differently on larger collections, but I'm going to guess not, and keep it like this then!